app/models/publication.rb

```diff
@@ -230,15 +230,19 @@ def year
   end
 
   def sciencewire_pub?
-    provenance == 'sciencewire'
+    provenance == Settings.sciencewire_source
   end
 
   def pubmed_pub?
-    provenance == 'pubmed'
+    provenance == Settings.pubmed_source
   end
 
   def wos_pub?
-    provenance == 'wos'
+    provenance == Settings.wos_source
+  end
+
+  def cap_pub?
+    provenance == Settings.cap_provenance
   end
```

Comment:
a new helper method for testing if this a manual publication by looking at the provenance

Reply:
so if `cap_pub?` is `true`, it's a manual publication? or if it's `false`, it's a manual publication? would you consider naming the method so that this is clearer? I'm thinking about the inebriation test.

Reply:
yeah, cap_pub? == true is a manual publication (basically just matching the string in the provenance) .... since its only used in one place in our new caluse, I can name the method something like "harvested_pub?", which will change the guard clause to a true instead of false